README.md

```diff
@@ -32,7 +33,9 @@ Available Commands:
 version     Output the version of prose
 
 Flags:
-    --dbsource string   Database source name; includes the DB driver as the scheme. E.g. postgres://user:password@localhost:5432/database?sslmode=disable
+    --dbsource string   Database source name; includes the DB driver as the scheme. Examples:
+                        postgres://user:password@localhost:5432/database?sslmode=disable
+                        mysql://user:password@tcp(localhost:3306)/database?tls=skip-verify
```

Comment:
This needs to match the code here: https://github.com/weaveworks/prometheus_sql_exporter/blob/master/cmd/root.go#L30